src/components/media-types/index.js

```diff
@@ -93,8 +59,8 @@ export const renderMediaType = ({
 
   displayView,
 }) => {
-  let parsedArtifactUri
-  let parsedDisplayUri
+  const parsedArtifactUri = artifactUri ? HashToURL(artifactUri) : ''
```

Comment:
i think the `HashToURL` function needs a second argument.

Reply:
By default it's optional and using the ENV key to choose the gateway, the second argument can be used to specify the gateway per instance use

Reply:
ah, true, i missed that 👍